scripts/log.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const winston = require('winston')
+/* eslint-disable no-console */
```

Comment:
We can put this in `scripts/.eslintrc` (in fact it should already be there in `master` if you rebase)